src/main.h

```diff
@@ -273,7 +280,8 @@ CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams);
  *
  * @param[out]   setFilesToPrune   The set of file indices that can be unlinked will be returned
  */
-void FindFilesToPrune(std::set<int>& setFilesToPrune, uint64_t nPruneAfterHeight);
+void FindFilesToPruneAuto(std::set<int>& setFilesToPrune);
+void FindFilesToPruneManual(std::set<int>& setFilesToPrune, int nManualPruneHeight);
```

Comment:
There are many functions in main that are not exported (see the whole namespace block for the handling of NodeState, for example).